muselog/datadog.py

```diff
@@ -23,7 +23,7 @@ def __init__(self, host: str, port: int):
         :param port: Datadog UDP input port
         """
 
-        DatagramHandler.__init__(self, host, port)
+        super().__init__(self, host, port)
```

Comment:
Does not need the `self` argument.

Reply:
My bad... I didn't test the call to `super` on the Datadog agent. I will modify and run against the agent as well with a test.